modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -83,6 +83,7 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultCodegen.class);
 
     public static FeatureSet DefaultFeatureSet;
+    public static final String LIBRARY_ONEOF_IMPL = "JavaSpring";
```

Comment:
You do not need this constant if you will move the code in SpringCodegen.java

Reply:
The same applies here as above